### DISCUSSION TITLE
Next 13 support

### DISCUSSION THREAD

**Discussion by tomjeannesson** [saltyshiomix/nextron#393] [Q&A]
Hi everyone,

Just started building a desktop app with nextron, and was wondering when we could expect official support for Next13.

Is it worth the time to fiddle with the code and set it up on my own? If so, how would I do it?

**Comment by akoptev** [comment]
Nextron with Next 13 would be rly nice. With example.

**Comment by saltyshiomix** [comment]
@tomjeannesson @akoptev 

Yesterday, I released nextron v9 which supports Next.js v13 and v14 🎉 

If you have time, please try it!

```
npx create-nextron-app your-nextron-app
```

**Reply by tomjeannesson** [reply]
Awesome! Thank you so much!
